crates/ruff_python_ast/src/nodes.rs

```diff
@@ -2735,6 +2735,21 @@ pub enum BoolOp {
     Or,
 }
 
+impl BoolOp {
+    pub fn as_str(&self) -> &'static str {
```

Comment:
Nit: any reason not to make this a `const fn`? (Same nit for `Operator::as_str`, `UnaryOp::as_str`, and `CmpOp::as_str`)

```suggestion
    pub const fn as_str(&self) -> &'static str {
```

Reply:
No reason but thanks for pointing it out.